### PR TITLE
Analyzer container size reduction + faster builds

### PR DESCRIPTION
**Pull request by omri374**
### Results:
- Full build time (with deps) down from 1:34 hours to 33 minutes
- Analyzer image size down from 3.5Gb to 1.7Gb

### Changes:
1. Changed base image from Alpine to slim-buster to support faster pip installs
2. Changed PIPENV commands to run faster
3. Avoiding PIP cache to reduce size
4. Updated spaCy model to 2.2.5 and updated unit tests accordingly
5. Updated Demo website text to support spaCy 2.2.5 with no errors
6. Ignore Deprecation Warnings when running PyTest

### CODE REVIEW

File: presidio-analyzer/analyzer/predefined_recognizers/us_bank_recognizer.py [hunk 1]
```diff
@@ -9,11 +9,11 @@
     "bank"
     # Task #603: Support keyphrases: change to "checking account"
     # as part of keyphrase change
-    "checking",
```
Comment (balteravishay) [inline presidio-analyzer/analyzer/predefined_recognizers/us_bank_recognizer.py]: is this relevant to this PR?

File: presidio-analyzer/analyzer/predefined_recognizers/us_passport_recognizer.py [hunk 2]
```diff
@@ -5,8 +5,9 @@
 # Weak pattern: all passport numbers are a weak match, e.g., 14019033
 VERY_WEAK_REGEX = r'(\b[0-9]{9}\b)'
 
+
 CONTEXT = [
-    "us", "united", "states", "passport", "number", "passport#", "travel",
```
Comment (balteravishay) [inline presidio-analyzer/analyzer/predefined_recognizers/us_passport_recognizer.py]: Is this relevant to this PR?

File: presidio-analyzer/analyzer/predefined_recognizers/us_bank_recognizer.py (hunk 1 above)
Comment (omri374) [reply presidio-analyzer/analyzer/predefined_recognizers/us_bank_recognizer.py]: Yes. I had to change the demo website text because spaCy now identifies new things (false positives). By changing this, I've encountered other issues with other recognizers...

File: presidio-analyzer/analyzer/predefined_recognizers/us_passport_recognizer.py (hunk 2 above)
Comment (omri374) [reply presidio-analyzer/analyzer/predefined_recognizers/us_passport_recognizer.py]: See previous comment
